pkgs/servers/teleport/default.nix

```diff
@@ -8,17 +8,17 @@
 , xdg-utils
 , nixosTests
 
-, withRoleTester ? true
+, withRoleTester ? !stdenv.isDarwin
```

Comment:
Since we skip tests on darwin we can save some build time as well. WDYT @justinas ?

Reply:
Role tester is not a unit test-only component (the name might have confused you). You can [read more about it here](https://goteleport.com/blog/testing-access-datalog/).

Looks like the darwin build was successful on ofborg. Do you have any additional information? I'm afraid I do not have a darwin machine to debug on.

Reply:
Thanks for the details. I'll try to reproduce the build failure and open a separate issue. (reverted the change)